Lib/test/_test_multiprocessing.py

```diff
@@ -2694,6 +2694,10 @@ def common(self, manager):
 
         self.assertEqual(foo.f(), 'f()')
         self.assertRaises(ValueError, foo.g)
+        with self.assertRaises(ValueError) as ctx:
+            foo._callmethod('g')
+        cause = ctx.exception.__cause__
+        self.assertIsInstance(cause, multiprocessing.pool.RemoteTraceback)
```

Comment:
Instead of testing low-level details, how about something pertaining to the user's experience?
For example you could have:
```python

class FooBar(object):
    def f(self):
        return 'f()'
    def g(self):
        raise ValueError  # exception raised here
    def _h(self):
        return '_h()'
```

And then `self.assertIn('# exception raised here', str(cause))`.

Reply:
Sounds good, but should I drop the `self.assertIsInstance(cause, multiprocessing.pool.RemoteTraceback)` check, then?